jobs/build/check-bugs/Jenkinsfile

```diff
@@ -63,8 +63,8 @@ node {
                 def out = sh(script: cmd.join(' '), returnStdout: true).trim()
                 echo out
 
-                if (out.contains('failed with')) {
-                    currentBuild.result = "FAILURE"
+                if (out.contains('Command failed:')) {
```

Comment:
`logging.info/warning` could print logs to stderr.

Reply:
I removed this, and now checking the return code of the script